packages/shared/src/components/tooltips/InteractivePopup.tsx

```diff
@@ -66,7 +66,7 @@ function InteractivePopup({
   position = InteractivePopupPosition.Center,
   closeOutsideClick,
   onClose,
-  closeButtonTypeClassname = 'btn-secondary',
+  closeButtonTypeClassname = ButtonVariant.Secondary,
```

Comment:
Should we change the name of the parameter as well? E.g. `closeButtonVariant` instead.

Reply:
We can do :) that's fine with me.
I'll adapt it.